.gitignore

```diff
@@ -1,2 +1,3 @@
 __pycache__
 *.pyc
+.DS_Store
```

Comment:
Maybe we should just include a standard complete Python `.gitignore` rather than the one off rules. 

Wanna add the full Python standard?

https://github.com/github/gitignore/blob/master/Python.gitignore

Reply:
Will still need to add `.DS_Store` but figure if we're updating might as well do it more comprehensively.